backend/app/routers/settings.py

```
@@ -291,6 +293,79 @@ def _get_provider_model(provider_name: str) -> Optional[str]:
+    try:
+        start = time.monotonic()
+        async with httpx.AsyncClient(timeout=10.0, follow_redirects=True) as client:
+            try:
+                response = await client.head(url)
+            except httpx.HTTPStatusError:
+                response = await client.get(url)
+        elapsed_ms = int((time.monotonic() - start) * 1000)
+
+        return URLTestResponse(
+            success=True,
+            message=f"URL可达，状态码: {response.status_code}",
```

Ruff (0.15.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)